conf_irods/partial_build.sh

```diff
@@ -2,19 +2,28 @@
 
 source env-files/use-local-irods.env
 
+echo "mkdir /root/.irods && cp /tmp/irods_environment_data.json /root/.irods/irods_environment.json" | $RUN_ON_DATA
```

Comment:
This looks like a bad merge conflict resolution too.  Maybe you could do a rebase to clear this stuff out.